### PR TITLE
chore: release 0.38.1

### PR DESCRIPTION
**Pull request by ThibaultFy**

### CODE REVIEW

**Comment by oleobal** [comment]
Is this supposed to include the algo to function rename ? 😅

**Comment by ThibaultFy** [comment]
> Is this supposed to include the algo to function rename ? 😅

Woooops ^^" sorry

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,18 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Remote test to verify that, with the right permission, we can test on an org from the training model of an other organization (#237)
 
 ### Changed
 
 - chore: rename tuple to task (#239)
-
-## [0.38.0] - 2023-01-30
-
-### Changed
-
```
Comment (oleobal) [inline CHANGELOG.md:21]: Are we supposed to remove this? I'm not sure as to the status of 0.38.0 but there is a GH release to that name so we should keep it in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,18 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.38.1] - 2023-06-02
```
Comment (oleobal) [inline CHANGELOG.md:10]:
```suggestion
## [0.38.1] - 2023-02-06
```

**Review by guilhem-barthes — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ThibaultFy) [reply CHANGELOG.md:21]: Yes you're right thanks ! :)

**Pull request merged**